modules/_labs/color-picker/react/stories/stories_VisualTesting.tsx

```diff
@@ -32,15 +34,13 @@ export const ColorPickerStates = () => (
             showCustomHexInput: true,
             resetColor: colors.blueberry400,
             resetLabel: 'Reset',
-            onColorReset: action('Reset Clicked'),
+            onColorReset: noop,
           },
         },
       ]}
       columnProps={[{label: 'Default', props: {}}]}
     >
-      {props => (
-        <ColorPicker {...props} transformOrigin={null} onColorChange={action('Color Changed')} />
-      )}
+      {props => <ColorPicker {...props} onColorChange={noop} />}
```

Comment:
fyi - this was failing without transformOrigin={null} originally because of the animation. If not needed, I'd love to learn why not

Reply:
I think per Lynn's comment of shipping the color picker swatch book with popup, the Popup animates into the frame, and Chromatic seems like it takes the snapshot before that happens. If we remove Popup from the color picker per: #836 this shouldn't be needed.

Reply:
Ah thanks for the context, I didn't know. It didn't seem necessary to me and it rendered fine in my local. Ran into the same trap as you (hit us up with an inline comment next time!)

There is a solution to this, I'll push it up in a sec. Basically Chromatic accounts for animations by pausing at the end. Unfortunately, I have to guess and check here, no way to really check this locally before we run snaps.

Reply:
FWIW, it's pretty rare we do this. In fact, we only do it in one other place: https://github.com/Workday/canvas-kit/blob/master/modules/modal/react/stories/stories_VisualTesting.tsx#L12-L16

Reply:
We're in business

![image](https://user-images.githubusercontent.com/14299381/90691867-f37f5900-e228-11ea-875d-f13b3bbf00f6.png)